hoot-test/src/main/cpp/hoot/test/ProcessPool.cpp

```diff
@@ -158,7 +158,13 @@ void ProcessThread::processJobs(JobQueue* queue)
           ++restart_count;
           //  Restart the process if there was a process failure
           if (restart_count < MAX_RESTART)
+          {
             _proc.reset(createProcess());
+            _outMutex->lock();
+            cout << test.toStdString() << " failed, requeued." << endl;
+            _outMutex->unlock();
+            output.clear();
+          }
```

Comment:
When a test crashes a process and is re-queued, the program outputs `<testname> failed, requeued.` and clears the output buffer of the stack trace if it is still under the `MAX_RESTART` limit.  If not the stack trace is printed and `HootTest` fails.